src/main/java/de/flapdoodle/embed/mongo/distribution/Version.java

```diff
@@ -223,7 +235,7 @@ public String toString() {
 
 		@Deprecated
 		LEGACY(V2_6),
-		PRODUCTION(V3_0),
+		PRODUCTION(V3_2),
```

Comment:
I added `V3_0 back`, but I don't fully understand how the `DEVELOPMENT` stuff is supposed to work. I think the newest development release is "3.2.1-rc3". Should I create a `V3_2_1_RC3` constant?